contracts/sol6/reserves/bridgeReserve/uniswap/KyberUniswapReserve.sol

```diff
@@ -0,0 +1,339 @@
+pragma solidity 0.6.6;
+
+import "../../../IKyberReserve.sol";
```

Comment:
maybe we can  have bridgeReserves as a separate folder. not inside reserves.
Seems to like we can have reserves as Kybers reserves (fpr, apr etc)
and bridgeReserves is a different thing.

but not sure.
lets discuss in next SC meeting.